webapps/frontend/app/viewmodels/wizards/addvdisk/data.js

```diff
@@ -19,7 +19,7 @@ define(['jquery', 'knockout', 'ovs/generic'], function($, ko, generic){
     var singleton = function() {
         var wizardData = {
             name:                  ko.observable('').extend({ regex: generic.vdiskNameRegex }),
-            sizeEntry:             ko.observable(0).extend({ numeric: { min: 1, max: 65535 } }),
+            sizeEntry:             ko.observable(0).extend({ numeric: { min: 1, max: 65535 }, throttle: 500}),
```

Comment:
Extend with `{ rateLimit: { method: "notifyWhenChangesStop", timeout: 500 } ` instead. This will only evaluate once the user stopped typing for 500ms